runtime/v1/shim/service.go

```diff
@@ -159,7 +159,7 @@ func (s *Service) Create(ctx context.Context, r *shimapi.CreateTaskRequest) (_ *
 			Source:  rm.Source,
 			Options: rm.Options,
 		}
-		if err := m.Mount(rootfs); err != nil {
+		if err = m.Mount(rootfs); err != nil {
```

Comment:
does this actually make a change? The function uses a named return variable, and the return on the line below returns an error, which will be assigned to that return variable

Reply:
@thaJeztah you are correct, the returned value will get used in the defer function

Reply:
Thanks for the comments.

I verified on https://play.golang.org/ that the above comment is correct.

But I think it is good practice to align the err variable which is assigned with what deferred func checks.

Reply:
In this case, the `err` is a local variable, and is basically discarded after use

```go
if err = m.Mount(rootfs); err != nil {
    # this returns a new variable;
    return nil, errors.Wrapf(err,
```

Having said the above; given that `err` is a very common name to use for local error variables, I tend to prefer named return variables clearly if they are important.

For example, name the named output variable `retErr` to make explicit in what parts of the code it is actually used.

Reply:
@thaJeztah 
I have updated the PR, can you take another look ?